gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
Before being able to merge this. Could you provide to an authoritative resource where I could check this script?

This will be executed in the developer environment, so I'm only inclined to accept standard copies, to avoid security issues. Hope you understand.

Same with the grandle.bat

Reply:
Of course, you can find the script here:
- https://github.com/gradle/gradle/blob/master/gradlew
- https://github.com/gradle/gradle/blob/master/gradlew.bat